Docs se16536 sqs dh (#862)
mulesoft/docs-connectors: amazon-sqs/5.6/modules/ROOT/pages/index.adoc

= Amazon SQS Connector 5.6 - Mule 4
:page-aliases: connectors::amazon/amazon-sqs-connector.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for Amazon SQS (Amazon SQS Connector) provides an easy way to interface with the Amazon Simple Queue Service (Amazon SQS) API, enabling Mule users to manage SQS queueing services without having to interact with the API directly.

> Amazon SQS offers a reliable, highly scalable hosted queue for storing messages as they travel between computers. By using Amazon SQS, developers can move data between distributed application components and perform different tasks without losing messages or requiring each component to be always available. Amazon SQS makes it easy to build an automated workflow that works closely with Amazon Elastic Compute Cloud (Amazon EC2) and other AWS infrastructure web services.

Reviewer: ```suggestion
Amazon SQS offers a reliable, highly scalable hosted queue for storing messages as they travel between computers. You can use Amazon SQS to move data between distributed application components and perform different tasks without losing messages or requiring each component to be always available. Amazon SQS makes it easy to build an automated workflow that works closely with Amazon Elastic Compute Cloud (Amazon EC2) and other AWS infrastructure web services.
```

== Audience

Read the Amazon SQS Connector documentation that meets the needs of the type of user that you are:

* Starting user
** To create your Mule app:
see xref:amazon-sqs-connector-design-center.adoc[Design Center Configuration]
or xref:amazon-sqs-connector-studio.adoc[Studio Configuration]. The xref:amazon-sqs-connector-config-topics.adoc[Additional Configuration Information]
topic helps extend your understanding of Amazon SQS Connector
** The xref:amazon-sqs-connector-examples.adoc[Examples] topic provides one or more use cases for using the connector.
* Power user
** Read the xref:amazon-sqs-connector-xml-maven.adoc[XML and Maven Support],
xref:amazon-sqs-connector-config-topics.adoc[Additional Configuration Information],
and xref:amazon-sqs-connector-examples.adoc[Examples] topics.

== Common Use Cases for the Connector

Common use cases for Amazon SQS Connector include sending messages to a queue and retrieving messages from a queue. For an example of these use cases, see xref:amazon-sqs-connector-examples.adoc#install-connector[Amazon SQS Examples].

== Next Step

After you have met the prerequisites and experimented with examples, you are ready to create an app with

> xref:amazon-sqs-connector-design-center.adoc[Design Center] or xref:amazon-sqs-connector-studio.adoc[Anypoint Studio].

Reviewer: I don't see the Design Center topic in the nav. I may have missed it. Just make sure to remove links to the Design Center topic if we removed that topic.
